src/main/java/org/sigmah/server/service/UserPermissionPolicy.java

```diff
@@ -228,4 +238,24 @@ public boolean isGranted(final OrgUnitProfile userOrgUnit, final GlobalPermissio
 		return false;
 	}
 
+	public boolean isGranted(List<OrgUnitProfile> userUnits, GlobalPermissionEnum permission) {
```

Comment:
I guess parameters should have been declared `final` to be consistent with above methods.

@Raphcal I know that you are a strong advocate of `final` everything (i like it), but when browsing the code base is not consistent. I was wondering, do you have a rule of when to declare things `final` or is it just that some guys did not followed the practice?